README.md

```diff
@@ -89,32 +89,34 @@ Next, clone this repository into a parent folder:
 
 ```
 (figshare_patrons) $ cd /path/to/parent/folder
-(figshare_patrons) $ git clone https://github.com/UAL-RE/ReQUIAM.git
+(figshare_patrons) $ git clone https://github.com/UAL-RE/ReQUIAM.git   (this is to clone a copy of the project to local directory) 
```

Comment:
Recommend keeping docs clean and concise. I.e., Any developer should know what `git clone` does